pkg/types/vars_source.go

```diff
@@ -48,6 +48,11 @@ type VarsSourceAwsSecretsManager struct {
 	Profile *string `json:"profile,omitempty"`
 }
 
+type VarsSourceGcpSecretsManager struct {
+	// Name of the secret. Should be provided in full resource url format: "projects/my-project/secrets/secret/versions/latest"
```

Comment:
your example path looks like a [relative resource name](https://cloud.google.com/apis/design/resource_names?hl=en#relative_resource_name) and not like a "full resource name" (I assume you meant this instead of "full resource url").